tools/clang/lib/SPIRV/DeclResultIdMapper.cpp

```diff
@@ -60,12 +60,23 @@ QualType getUintTypeWithSourceComponents(const ASTContext &astContext,
                    "getUintTypeWithSourceComponents");
 }
 
-uint32_t getLocationCount(const ASTContext &astContext, QualType type) {
+uint32_t getLocationCount(const ASTContext &astContext, QualType type,
+                          bool rowMajorAttr) {
```

Comment:
I don't think `bool rowMajorAttr` should be passed to this function. Most types are not matrices.

I think `spirvOptions` should be passed to this function.